packages/bierzo-wallet/.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
Why do you need this?

Reply:
If the .env files are removed, `yarn start` fails. It's a temporary fix that I found so that I could continue implementing the test.

There is some kind of dependency problem that I don't fully understand. I believe it has to do with different dependencies needing different versions of Jest. I also had to upgrade react-scripts from 2.1.8 to 3.0.0 because of the issues that started to appear.

Reply:
updating the scripts into the 3.0 version is dangerous, I will do that in the next Friday of code. Thanks for the explanation, tomorrow I will look into it.